src/routes/auth/new-user/+page.svelte

```diff
@@ -0,0 +1,94 @@
+<script>
+// @ts-nocheck
+
+    import { page } from "$app/stores";
+    import welcome from '$lib/images/svelte-welcome.png'
+    import { ADD_NEW_USER } from "$lib/mutations/users";
+    import { onMount } from "svelte";
+  import { mutation } from "svelte-apollo";
+
+    let username = '';
+    let name = '';
+    let email = '';
+
+    onMount( () => {
+    page.subscribe((value) => {
+        const {user} = value.data.session;
+        name = user.name;
+        email = user.email;
+        username = user.username;
+    })
+    })
+
+    const addNewUser = mutation(ADD_NEW_USER)
+
+    async function handleSubmit() {
+        email.toLowerCase
+        try {
+            const result = await addNewUser({
+                variables: {
+                    email,
+                    name,
+                    username,
+                }
+            })
+
+            if (result.data.insert_users_one) {
+                console.log('Mutation successful!');
+                window.location.href = `/`;
+            } else {
+                console.log('Mutation failed');
+            }
+            
+        } catch (error) {
+            console.log(error)
+        }
+    }
+
+</script>
+
+
+<div class=" h-screen grid grid-cols-1 place-items-center md:grid-cols-2">
+    <img src={welcome} alt="Welcome" />
+
+
+    <form class="form-control gap-4 w-full max-w-xs">
+        <div>
+            <label for="username" class="label">
+                <span class="label-text">Username</span>
+            </label>
+            <input 
+                id="username" type="text" 
+                bind:value={username} 
+                on:input={(event)=> username = event.target.value.toLowerCase()} 
+                class="input input-bordered w-full max-w-xs"
+                required
+                />
+        </div>
+        <div>
+            <label for="name" class="label">
+                <span class="label-text">Full Name</span>
+            </label>
+            <input 
+                id="name" 
+                type="text" 
+                bind:value={name} 
+                class="input input-bordered w-full max-w-xs" 
+                required
+                />
+        </div>
+        <div>
+            <label for="email" class="label">
+                <span class="label-text">Email Address</span>
```

Comment:
The email address field might be redundant, as this is often provided by 0Auth

Reply:
I had an issue where my email was passes as undefined from github provider

Reply:
I guess we could render this field conditionally because it would be redundant to include it if an email is readily available, specially when using Google as an auth provider.